frontend/views/components/Modal/Modal.vue

```diff
@@ -35,31 +38,38 @@ export default {
   },
   watch: {
     '$route' (to, from) {
-      if (to.query.modal) {
-        // We reset the modals with no animation for simplicity
-        if (to.query.modal !== this.content) this.content = to.query.modal
-        const subcontent = to.query.subcontent
-        if (subcontent !== this.activeSubcontent()) {
-          // Try to find the new subcontent in the list of subcontent
-          const i = this.subcontent.indexOf(subcontent)
-          if (i) {
-            this.subcontent = this.subcontent.splice(0, i)
-          } else this.subcontent = subcontent
+      if (this.isUrlChange) {
+        console.log('URLCHANGE')
```

Comment:
For the PR to be merged into `master`, any debug logging should be either removed, or be logged with `console.debug`, and made clearer about what it is logging (just saying "URLCHANGE" is clear only to you, but it needs to be clear to other developers as well: it needs to tell them which file/function is logging this, and why it's being logged)